src/main/webapp/app/entities/exercise.model.ts

```diff
@@ -236,3 +236,7 @@ export function resetDates(exercise: Exercise) {
     exercise.assessmentDueDate = undefined;
     exercise.exampleSolutionPublicationDate = undefined;
 }
+
+export function isNotReleased(exercise: Exercise) {
+    return exercise.isAtLeastTutor && exercise.releaseDate != undefined && !dayjs(exercise.releaseDate).isBefore(dayjs());
```

Comment:
Can you add a documentation to this function? Especially the `isAtLeastTutor` confuses me.

Reply:
I used the logic from the unreleased tag. But after thinking about this, the isAtLeastTutor flag is actually unnecessary.
It's now simplified! 👍